packages/editor/rich-text-editor/src/ui/menus/bubble-menu/index.tsx

```diff
@@ -125,6 +113,7 @@ export const EditorBubbleMenu: FC<EditorBubbleMenuProps> = (props: any) => {
             editor={props.editor!!}
             isOpen={isLinkSelectorOpen}
             setIsOpen={() => {
+              console.log("setIsLinkSelectorOpen");
```

Comment:
remove debug console logs

Reply:
done in https://github.com/makeplane/plane/pull/2789/commits/7085e6e2f6692005aede682344d1d6d423535713